common/djangoapps/student/models_api.py

```diff
@@ -62,3 +63,27 @@ def get_phone_number(user_id):
         log.exception(exception)
         return None
     return student.phone_number or None
+
+
+def get_course_access_role(user, org, course_id, role):
+    """
+    Get a specific CourseAccessRole object. Return None if
+    it does not exist.
+
+    Arguments:
+        user: User object for the user who has access in a course
+        org: the org the course is in
+        course_id: the course_id of the CourseAccessRole
+        role: the role type of the role
+    """
+    try:
+        course_access_role = _CourseAccessRole.objects.get(
+            user=user,
+            org=org,
+            course_id=course_id,
+            role=role,
+        )
+    except CourseAccessRole.DoesNotExist as exception:
+        log.exception(exception)
```

Comment:
I think it make sense to improve the exception message here, so it would be easier to search in splunk for it.